allure-commandline/src/main/java/io/qameta/allure/CommandLine.java

```diff
@@ -75,6 +75,10 @@ public static void main(final String[] args) throws InterruptedException {
 
     @SuppressWarnings("PMD.AvoidLiteralsInIfCondition")
     public Optional<ExitCode> parse(final String... args) {
+        if (args.length == 0) {
```

Comment:
I guess no need to check arguments length at this point. `ParameterException` is thrown for all the unknown options and commands, am I right?